app/javascript/dashboard/store/modules/conversations/getters.js

```diff
@@ -18,24 +19,30 @@ const getters = {
     );
     return selectedChat || {};
   },
-  getMineChats(_state) {
+  getMineChats: _state => activeFilters => {
     const currentUserID = authAPI.getCurrentUser().id;
-    return _state.allConversations.filter(chat =>
-      !chat.meta.assignee
-        ? false
-        : chat.status === _state.chatStatusFilter &&
-          chat.meta.assignee.id === currentUserID
-    );
+
+    return _state.allConversations.filter(chat => {
+      const hasAssignee = !!chat.meta.assignee;
+      const isAssignedToMe = chat.meta.assignee.id === currentUserID;
+      const shouldFilter = applyPageFilters(chat, activeFilters);
+      const isChatMine = hasAssignee && isAssignedToMe && shouldFilter;
+
+      return isChatMine;
+    });
   },
-  getUnAssignedChats(_state) {
-    return _state.allConversations.filter(
-      chat => !chat.meta.assignee && chat.status === _state.chatStatusFilter
-    );
+  getUnAssignedChats: _state => activeFilters => {
```

Comment:
I would suggest to rename the getters to use conversation instead of chat

Reply:
Will take another PR to refactor #1952